src/types/index.ts

```diff
@@ -16,6 +16,7 @@ export type TextRequest = string;
 
 export interface NotionToMarkdownOptions {
   notionClient: Client;
+  convertImagesToBase64?: boolean;
```

Comment:
please sync with the latest changes in the master.

https://github.com/souvikinator/notion-to-md/blob/5cd0b5e1d596d7d5ddfd4698582b196c5912c636/src/types/index.ts#L22